src/api/CoinBase.ts

```diff
@@ -27,33 +28,28 @@ const CURRENCY_MAPPING: Record<string, string> = {
 
 export default class CoinBase extends BaseExchangeApi {
     constructor() {
-        super('CoinBase', CURRENCY_MAPPING, ['RIF']);
+        super('CoinBase', CURRENCY_MAPPING, ['RIF', 'RBTC']);
     }
 
     async queryExchangeRate(
         sourceCurrency: string,
         targetCurrency: string
     ): Promise<BigNumber> {
         const upperCaseTargetCurrency = targetCurrency.toUpperCase();
-        const response: Response = await fetch(
-            `${URL}?currency=${sourceCurrency}`,
-            {
-                method: 'get'
-            }
-        );
+        let response: AxiosResponse;
 
-        if (!response.ok) {
-            const {
-                errors: [error]
-            }: CoinBaseErrorResponse = await response.json();
+        try {
+            response = await axios.get(`${URL}?currency=${sourceCurrency}`);
+        } catch (error: unknown) {
+            const { response } = error as AxiosError;
             throw Error(
-                `CoinBase API status ${response.status}/${response.statusText}/${error.message}`
+                `CoinBase API status ${response.status}/${response.statusText}`
```

Comment:
I think it is possible to error without response:
```suggestion
                `CoinBase API status ${response?.status}/${response?.statusText}`
```
See: https://github.com/rsksmart/rif-relay-client/blob/45337c5a7c3958dcee5610c77a0054ae43606b11/src/api/common/HttpWrapper.ts#L40

Ref: https://axios-http.com/docs/handling_errors

Reply:
Done. 
However, once we migrate this to support ethers, we may need to unify the error handling to avoid code repetition.